Android/GoogleDagger/source/Dagger/Transforms/Metadata.xml

```diff
@@ -1,4 +1,8 @@
 ﻿<?xml version="1.0" encoding="UTF-8"?>
 <metadata>
-	<attr path="/api/package[@name='com.google.dagger.dagger']" name="managedName">Google.Dagger</attr>
+    <attr path="/api/package[@name='dagger.internal']/class[@name='MapFactory']/method[@name='get' and count(parameter)=0]" name="managedName">GetDictionary</attr>
+    <attr path="/api/package[@name='dagger.internal']/class[@name='MapProviderFactory']/method[@name='get' and count(parameter)=0]" name="managedName">GetDictionary</attr>
+    <attr path="/api/package[@name='dagger.internal']/class[@name='SetFactory']/method[@name='get' and count(parameter)=0]" name="managedName">GetCollection</attr>
+    <attr path="/api/package[@name='dagger.internal']/class[@name='ProviderOfLazy']/method[@name='get' and count(parameter)=0]" name="managedName">GetLazy</attr>
+    <remove-node path="/api/package[@name='dagger.internal']/class[@name='MapProviderFactory']/implements" />
```

Comment:
I see you remove this here, and then add it back in the Classes.cs. Is there a reason?

Reply:
Because there are two interfaces with the same method signature. 

All of the method name changes prior to it couldn't fix the issue. 
The other does fix because it only implement single interface or the method signature is different.

Cheers.